command/operator_debug_test.go

```diff
@@ -32,6 +32,29 @@ func TestDebugUtils(t *testing.T) {
 	require.Equal(t, "https://127.0.0.1:8500", e.addr("foo"))
 }
 
+func TestDebugSuccesses(t *testing.T) {
+	t.Parallel()
+	srv, _, _ := testServer(t, false, nil)
+	defer srv.Shutdown()
+
+	ui := new(cli.MockUi)
```

Comment:
As per https://github.com/hashicorp/nomad/pull/9027: 
```suggestion
	ui := cli.NewMockUi()
```

Reply:
fixed